package.json

```diff
@@ -36,39 +36,40 @@
     "release": "standard-version"
   },
   "dependencies": {
+    "@jest/test-sequencer": "^25.4.0",
     "ansi-html": "0.0.7",
     "bonjour": "^3.5.0",
     "chokidar": "^3.4.0",
     "compression": "^1.7.4",
     "connect-history-api-fallback": "^1.6.0",
     "debug": "^4.1.1",
-    "del": "^4.1.1",
+    "del": "^5.1.0",
     "express": "^4.17.1",
     "html-entities": "^1.3.1",
-    "http-proxy-middleware": "0.19.1",
-    "import-local": "^2.0.0",
-    "internal-ip": "^4.3.0",
+    "http-proxy-middleware": "^1.0.3",
+    "import-local": "^3.0.2",
+    "internal-ip": "^6.0.0",
     "ip": "^1.1.5",
     "is-absolute-url": "^3.0.3",
     "killable": "^1.0.1",
     "loglevel": "^1.6.8",
     "open": "^7.0.3",
-    "p-retry": "^3.0.1",
+    "p-retry": "^4.2.0",
     "portfinder": "^1.0.25",
-    "schema-utils": "^1.0.0",
+    "schema-utils": "^2.6.6",
     "selfsigned": "^1.10.7",
-    "semver": "^6.3.0",
+    "semver": "^7.3.2",
     "serve-index": "^1.9.1",
     "sockjs": "0.3.20",
     "sockjs-client": "1.4.0",
     "spdy": "^4.0.2",
-    "strip-ansi": "^3.0.1",
-    "supports-color": "^6.1.0",
+    "strip-ansi": "^6.0.0",
```

Comment:
We can't do it, we will break client on old browsers, we need setup babel on our client and then do update (let's do backport it from previously next)